sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/src/Generated/Models/GuestConfigurationAssignmentProperties.cs

```diff
@@ -55,13 +57,14 @@ public GuestConfigurationAssignmentProperties()
         /// <param name="resourceType">Type of the resource - VMSS / VM</param>
         /// <param name="vmssVMList">The list of VM Compliance data for
         /// VMSS</param>
-        public GuestConfigurationAssignmentProperties(string targetResourceId = default(string), GuestConfigurationNavigation guestConfiguration = default(GuestConfigurationNavigation), string complianceStatus = default(string), System.DateTime? lastComplianceStatusChecked = default(System.DateTime?), string latestReportId = default(string), AssignmentReport latestAssignmentReport = default(AssignmentReport), string context = default(string), string assignmentHash = default(string), string provisioningState = default(string), string resourceType = default(string), IList<VMSSVMInfo> vmssVMList = default(IList<VMSSVMInfo>))
+        public GuestConfigurationAssignmentProperties(string targetResourceId = default(string), GuestConfigurationNavigation guestConfiguration = default(GuestConfigurationNavigation), string complianceStatus = default(string), System.DateTime? lastComplianceStatusChecked = default(System.DateTime?), string latestReportId = default(string), string parameterHash = default(string), AssignmentReport latestAssignmentReport = default(AssignmentReport), string context = default(string), string assignmentHash = default(string), string provisioningState = default(string), string resourceType = default(string), IList<VMSSVMInfo> vmssVMList = default(IList<VMSSVMInfo>))
```

Comment:
As interface changes caused not backward compatible, please increate the major version of the package, not minor version

Reply:
done